elements/pfelement/README.md

```diff
@@ -20,11 +20,21 @@ class MyElement extends PFElement {
     return "my-element.scss";
   }
 
+  // Alternative: Embed the styles directly in the JS.
```

Comment:
@castastrophe Can we get rid of this alternative? I'd really like to continue with using just scss files instead of suggesting this alternative.

Reply:
I'm not sure it necessarily hurts to make both available.  One example is a component with only styles for `:host([hidden]) { display: none; }`.  Does that need Sass and a separate file at that point?  Maybe not yet.

Reply:
These are the styles in the pfe-number.scss file:
```scss
:host {
  display: inline;
  white-space: nowrap;
}
```

Reply:
One more example from pfe-datetime.scss:
```scss
:host {
    display: inline;
}
```

Reply:
I just thought of something else cool you could use styles for:
```js
get styles() {
   return `<link rel="stylesheet" href="https://mycdn/external-styles.css">`;
}
```